fastlane/lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -37,6 +37,10 @@ def self.run(params)
         changelog = changelog.gsub("\n\n", "\n") if changelog # as there are duplicate newlines
         Actions.lane_context[SharedValues::FL_CHANGELOG] = changelog
 
+        puts ""
+        puts changelog
```

Comment:
Ah, very good point, thanks for letting me know, let's leave it like this then 👍